.github/workflows/ci.yml

```
@@ -291,6 +291,34 @@ jobs:
           path: ${{ env.RUST_BIN_DIR }}/${{ matrix.binary }}
           retention-days: 1
 
+  cargo-vet:
+    name: Vet Dependencies
+    runs-on: ubuntu-latest
```

@sehz to scope this only to `staging` perhaps we could do:

```suggestion
  cargo-vet:
    name: Vet Dependencies
    runs-on: ubuntu-latest
    if: github.event_name == 'push' && github.ref == 'refs/heads/staging'
```

This way we will only run when pushing to the `staging` branch.

---

yes that make sense